docs/content/documentation/content/overview.md

```diff
@@ -23,30 +23,30 @@ that contains [pages](@/documentation/content/page.md): your `.md` files.
         └── _index.md // -> https://mywebsite.com/landing/
 ```
 
-Each page path (the part after the `base_url`, for example `blog/cli-usage/`) can be customised by changing the `path` or `slug`
-attribute of the [page front-matter](@/documentation/content/page.md#front-matter).
+Each page path (the part after `base_url`, for example `blog/cli-usage/`) can be customised by changing the `path` or
+`slug` attribute of the [page front-matter](@/documentation/content/page.md#front-matter).
 
 You might have noticed a file named `_index.md` in the example above.
-This file is used to store both metadata and content of the section itself and is not considered a page.
+This file is used to store both the metadata and content of the section itself and is not considered a page.
 
-To make sure the terminology used in the rest of the documentation is understood, let's go over the example above.
+To ensure that the terminology used in the rest of the documentation is understood, let's go over the example above.
 
 The `content` directory in this case has three `sections`: `content`, `blog` and `landing`. The `content` section has only
-one page, `something.md`, the `landing` section has no page and the `blog` section has 4 pages: `cli-usage.md`, `configuration.md`, `directory-structure.md`
-and `installation.md`.
+one page (`something.md`), the `landing` section has no pages and the `blog` section has 4 pages (`cli-usage.md`,
+`configuration.md`, `directory-structure.md` and `installation.md`).
 
-While not shown in the example, sections can be nested indefinitely.
+Sections can be nested indefinitely.
 
-## Assets colocation
+## Asset co-location
```

Comment:
same as above, I'm not sure breaking the links is worth it if they are both correct

Reply:
I agree. Please revert.